app/views/surveys/_show-single-survey.erb

```diff
@@ -0,0 +1,8 @@
+<div class='sinlge-survey'>
+  <h2> survey.title </h2>
+  <%= survey.questions %>
```

Comment:
I think you'll have to do a "if survey.questions", then questions.each do, send to question partial